api/CharacterData.json

```
@@ -391,6 +489,55 @@
           }
         }
       },
+      "remove": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CharacterData/remove",
+          "spec_url": "https://dom.spec.whatwg.org/#ref-for-dom-childnode-remove①",
+          "support": {
+            "chrome": {
+              "version_added": "23"
```

```suggestion
              "version_added": "24"
```

This is from mdn-bcd-colllector. chrome_android, opera and samsung don't need fixing.